config/locales/en/helpers.yml

```diff
@@ -307,6 +311,8 @@ en:
       steps_check_under_age_form:
         caution: How old were you when you got cautioned?
         conviction: How old were you when you got convicted?
+      steps_check_caution_or_conviction_form:
+        add_caution_or_conviction: Do you have another caution on conviction you'd like to check?
```

Comment:
typo, usage of `on` between word `caution` and word `conviction`. Meant to be `or`